src/earn/EarnActivePool.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import SkeletonPlaceholder from 'src/components/SkeletonPlaceholder'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { fetchAavePoolInfo } from 'src/earn/poolInfo'
+import UpwardGraph from 'src/icons/UpwardGraph'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import Logger from 'src/utils/Logger'
+import networkConfig from 'src/web3/networkConfig'
+import { Address } from 'viem'
+
+const TAG = 'earn/EarnActivePool'
+
+function PoolDetailsLoading() {
+  return (
+    <View>
+      <SkeletonPlaceholder backgroundColor={Colors.gray2} highlightColor={Colors.white}>
+        <View
+          style={{
+            ...typeScale.labelXSmall,
+            width: 64,
+            borderRadius: 100,
+          }}
+        />
+      </SkeletonPlaceholder>
+    </View>
+  )
+}
+
+export default function EarnActivePool() {
+  const { t } = useTranslation()
+  const token = useTokenInfo(networkConfig.arbUsdcTokenId)
+  const poolToken = useTokenInfo(networkConfig.aaveArbUsdcTokenId)
+  const asyncPoolInfo = useAsync(async () => {
+    if (!token || !token.address) {
```

Comment:
if you use the viem type guard here for `Address` `|| !isAddress(token.address)` you can avoid the type assertion down below

Reply:
Fixed in f5b7c8e7e7335ddfc9005344dfec0dc7727ca36d!